server-api/src/main/java/org/apache/atlas/RequestContext.java

```diff
@@ -403,6 +405,14 @@ public void setPoliciesBootstrappingInProgress(boolean policiesBootstrappingInPr
         isPoliciesBootstrappingInProgress = policiesBootstrappingInProgress;
     }
 
+    public boolean isAccessControlMigrationInProgress() {
```

Comment:
Lets avoid this check as discussed. In general, if a user is removed and the policy exists, (approach I can think off) let us use current admin ( which should exists for sure ) and move forward.

Reply:
Reverted this skip check, this will be handled by migration script, it will fallback to connection creator user to impersonate